modules/agent-installer-architectures.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/preparing-to-install-with-agent-based-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="agent-install-networking_{context}"]
+= Optional: Verifying architectures on the Agent-based installer 
+
+You can check which architecture your {product-title} agent installer cluster is using by viewing the version binary.
+
+.Prerequisites: 
+
+* You installed the OpenShift CLI (oc)
+
+.Procedure: 
+
```

Comment:
Get the release payload from the openshift-installer:
$ ./openshift-install version
then determine if the release payload supports multiple architectures:

$ oc adm release info quay.io/openshift-release-dev/ocp-release@sha256:27d41f99e0784a2f596e773799983147b794df48d1f7536d51a95a60945b1931 -o jsonpath="{ .metadata.metadata}"

If the above returns {"release.openshift.io/architecture":"multi"}, then this installer will support arm, power, s390x, and x86.

Otherwise you can only install the architecture defined for "release architecture" outputted from  `openshift-install version`